### ISSUE TITLE
Keep website alive in background

### ISSUE THREAD

**Issue by 00sapo** [baotlake/anything-copilot#1]
As of now, whenever you close the side panel, the connection the the websites are closed. When you reopen it, websites are reloaded.

For some websites, this is annoying, e.g. web.whatsapp.com

**Comment by baotlake** [comment]
Yes, the page needs to be reloaded every time after closing, which can be frustrating. Unfortunately, it appears there's no immediate fix available.
